cmd/emp/main.go

```diff
@@ -103,6 +104,13 @@ func (c *Command) ShortExtra() string {
 	return c.Short[:len(c.Short)-len(extra)]
 }
 
+func (c *Command) CheckNumArgs(args []string) {
+	if len(args) != c.NumArgs {
```

Comment:
Instead of having to explicitly set `NumArgs`, we could parse this out of `Usage`, since it follows the convention `subcommand <arg1> <arg2> <etc>`.